Graphs/GaugeGraph/gaugeGraph.test.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import { getDataAndConfig } from '../testHelper';
+import GaugeGraph from '.';
+
+const cheerio = require('cheerio');
+
+describe("GaugeGraph", () => {
+    let config;
+    beforeAll(async () => {
+        config = await getDataAndConfig('GaugeGraph');
+    });
+
+    describe("Initial Configurations", () => {
+        let gaugeGraph, $;
+        beforeAll((done) => {
+            const element = document.createElement("div");
```

Comment:
@mayank-nx Move this element to outside, as it might not be accessible below.

Reply:
@nxanil  Suggested change has been done.